godtools/App/Services/Renderer/Views/Tool/Views/Modal/ToolPageModalView.swift

```diff
@@ -98,9 +99,9 @@ class ToolPageModalView: MobileContentView {
         centerContentStackVerticallyIfNeeded()
     }
     
-    override func didReceiveEvents(eventIds: [MultiplatformEventId]) {
+    override func didReceiveEvents(eventIds: [EventId]) {
             
-        let eventsContainFollowUp: Bool = eventIds.contains(MultiplatformEventId.followUp)
+        let eventsContainFollowUp: Bool = eventIds.contains(EventId.Companion().FOLLOWUP)
```

Comment:
are you triggering the modal if the FOLLOWUP event is triggered? There is actually listeners defined on the Modal model that contain which events should trigger a specific modal.

Reply:
hmm, I see you're using the listeners on line 108, but I'm confused why it's filtering based on the presence of the followup event.

Reply:
Levi is going to investigate this in a followup ticket: https://jira.cru.org/browse/GT-1480